BuildResidentialScheduleFile/resources/schedules.rb

```diff
@@ -345,7 +345,6 @@ def create_stochastic_schedules(args:)
     lighting_sch = schedule_config['lighting']
     ceiling_fan_sch = schedule_config['ceiling_fan']
 
-    monthly_lighting_schedule = schedule_config['lighting']['monthly_multiplier']
```

Comment:
@rajeee This variable was not being used. Is this a bug (i.e., it was supposed to be used) or is it intentional that it isn't used?

Reply:
I found that the lighting monthly multipliers are being applied elsewhere in the code. So there is no bug here.